userprefs.php

```diff
@@ -268,7 +268,7 @@ function echo_general_tab() {
         _('E-mail'), 'email', 'email',
```

Comment:
I have just noticed that we do not check if `email` is actually a valid email address, for example this is considered valid:

![image](https://user-images.githubusercontent.com/3049847/52418149-b98aae00-2aed-11e9-8bb4-1e12b67ca1ed.png)

Maybe we should at least use [`<input type="email" ...>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email)?

Reply:
Oh nice! I didn't know about that either. I've added that for userprefs and for account registration.